lib/spack/docs/pipelines.rst

```diff
@@ -169,11 +169,25 @@ have disabled it (using ``rebuild-index: False``) because the index would only b
 generated in the artifacts mirror anyway, and consequently would not be available
 during subesequent pipeline runs.
 
+.. note::
+   With the addition of reproducible builds (#22887) a previously working
+   pipeline will require some changes:
+   - In the build jobs (`runner-attributes`), the environment location changed.
```

Comment:
In restructured text, I think you have to use double backticks instead of single.

Reply:
Done.